packages/xo-server/src/api/remote.js

```diff
@@ -42,30 +42,32 @@ list.params = {
   id: { type: 'string' },
 }
 
-export async function create({ name, url, options }) {
-  return this.createRemote({ name, url, options })
+export function create(props) {
+  return this.createRemote(props)
 }
 
 create.permission = 'admin'
 create.description = 'Creates a new fs remote point'
 create.params = {
   name: { type: 'string' },
-  url: { type: 'string' },
   options: { type: 'string', optional: true },
+  proxyId: { type: 'string', optional: true },
+  url: { type: 'string' },
 }
 
-export async function set({ id, name, url, options, enabled }) {
-  await this.updateRemote(id, { name, url, options, enabled })
+export async function set({ id, ...props }) {
```

Comment:
I don't understand why you destructure props here ?

Reply:
I destruct the params to use `rest`.
I use `rest` to simplify the code and because the properties are already documented.